firefox_puppeteer/ui/toolbars.py

```diff
@@ -446,3 +446,21 @@ def verifier(self):
         :returns: Reference to the identity-popup content verifier.
         """
         return self.marionette.find_element(By.ID, 'identity-popup-content-verifier')
+
+    def close(self, force=False):
+        """Closes the identity popup.
+
+        :param force: If true, the popup is closed by its own hide function,
+                      otherwise a key event is sent to close the popup.
```

Comment:
I would shorten the description to "If True, the popup will be force closed". The second part you can add to the method summary like "... popup by hitting the escape key".